lib/ReactViews/DataCatalog/DataCatalogItem.jsx

```diff
@@ -54,21 +53,19 @@ export const DataCatalogItem = observer(
     },
 
     toggleEnable(event) {
-      runInAction(() => {
+      runInAction(async () => {
```

Comment:
`runInAction(async` is never a good mix. There are some good ideas to fix this here: https://mobx.js.org/actions.html#asynchronous-actions

Reply:
Yep I forgot about that, thanks for picking it up - `runInAction` wasn't needed here so I've removed it